samples/browser/vitest.config.ts

```diff
@@ -24,5 +22,5 @@ export default defineConfig(async () => {
         ],
       }
     },
-  }
+  } satisfies ViteUserConfigExport;
```

Comment:
This was a little bit cursed, `any` alone wasn't telling ts that this was a valid argument to `defineConfig(…)` 🤷